### ISSUE TITLE
Fatal error when using prebind

### ISSUE THREAD

**Issue by andrei-pmbcn** [conversejs/converse.js#1502]
I am using prebind authentication. When I initialize converse, I receive the error "FATAL: TypeError: Cannot read property 'insertAdjacentElement' of null" that can be traced back to converse-register.js:57, where this.el.querySelector('.buttons') is set to null because, presumably, the querySelector function cannot find any elements. This error occurs when using prebind authentication but not login authentication.

I am writing it here on github because the lack of a clear error message that describes the problem on the user's end is something that ought to be resolved.

**Comment by jcbrand** [comment]
Did you modify any templates or change the code in any way?

**Comment by andrei-pmbcn** [comment]
No, I have used a fresh install of converse and have included converse.min.js in the html file. Here is the converse.initialize script:
```
converse.initialize({
    locales_url: 'apps/view/javascript/converse/locale/ro/LC_MESSAGES/converse.json',
    bosh_service_url: 'http://localhost:5443/bosh',
    debug: true,
    prebind_url: 'http://localhost/index.php?[query string not shown for security reasons]',
    authentication: 'prebind',
    allow_logout: false,
    keepalive: true,
    message_carbons: true,
    play_sounds: true,
    roster_groups: true,
    show_controlbox_by_default: true,
    xhr_user_search: false
});
```

Note that I am also getting another error afterwards:
```
Uncaught (in promise) Error: restoreBOSHSession: tried to restore a "keepalive" session but we don't have the JID for the user!
    at Object.restoreBOSHSession (converse-core.js:1059)
    at Object.attemptPreboundSession (converse-core.js:1081)
    at Object.logIn (converse-core.js:1183)
    at D (converse-core.js:423)
    at Object.T.initialize (converse-core.js:1226)
```

**Comment by chvanikoff** [comment]
Having the same errors. Any update on this?

**Comment by jcbrand** [comment]
I believe this is fixed in a46ee4dfe1
